tests/yam/agama/boot_encrypted.pm

```diff
@@ -0,0 +1,44 @@
+## Copyright 2024 SUSE LLC
+# SPDX-License-Identifier: GPL-2.0-or-later
+
+# Summary: Boot the system for encryption scenario.
+# - {Enter passphrase for root partition for full disk encryption}.
+# - Select first entry to boot in grub2
+# - {Enter passphrase for swap partition for full disk encryption}.
+# = {Enter passphrase for home partition for home encryption}.
+# Maintainer: QE YaST and Migration (QE Yam) <qe-yam at suse de>
+
+use base 'opensusebasetest';
+use strict;
+use warnings;
+use testapi;
+
+sub run {
+    my $enter_passphrase_for_root = $testapi::distri->get_enter_passphrase_for_root();
+    my $enter_passphrase_for_swap = $testapi::distri->get_enter_passphrase_for_swap();
+    my $enter_passphrase_for_home = $testapi::distri->get_enter_passphrase_for_home();
+    my $grub_menu = $testapi::distri->get_grub_menu_installed_system();
+
+    if (check_var('FULL_LVM_ENCRYPT', '1')) {
```

Comment:
my suggestion was different, to use old settings that we actually want to remove in some point is not a good idea, a new one with the partitions you expect would be much more clear.

Reply:
This module is for qemu, please review it. And for remote architecture, create a separate module?

Reply:
let's postpone other architecture and use only qemu here, maybe when the PR would be looking mergeable we can decide. Please exclude code code that is not qemu a provide fresh verification in first comment.